news/amd.rst

```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* Functions to compute amd and pdd in src/diffpy/metrics/amd.py
+* Tests for amd and pdd functions in tests/test_amd.py
```

Comment:
please remove mention of tests here.  It is assumed that tests will be written when new features are added